nautobot_golden_config/api/views.py

```diff
@@ -91,3 +94,23 @@ class ConfigReplaceViewSet(CustomFieldModelViewSet):  # pylint:disable=too-many-
     queryset = models.ConfigReplace.objects.all()
     serializer_class = serializers.ConfigReplaceSerializer
     filterset_class = filters.ConfigReplaceFilterSet
+
+
+class ConfigPushPermissions(BasePermission):
```

Comment:
This should be documented in the ADR, I don't know anywhere else we do this.

Reply:
resolved in https://github.com/nautobot/nautobot-plugin-golden-config/commit/5b88c8afa091542b049f6de527ecb0d4001df50b